src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1429,6 +1523,7 @@ asf::auto_release_ptr<asr::Project> build_project(
     const FrameRendParams&                  frame_rend_params,
     const RendererSettings&                 settings,
     Bitmap*                                 bitmap,
+    RendProgressCallback*                   progress_cb,
```

Comment:
Always make `progress_cb` the last argument, for all functions (it's already the untold convention in the rest of the code).